docs/integrations/overview.md

```diff
@@ -320,23 +325,177 @@ After that, user receives a withdrawal voucher that they can use to withdraw the
 ### Withdrawal
 
 Withdrawal of the ASSET implies sending the NFT to `withdrawal_manager` with the following message attached:
+
+```json
+{
+  "withdraw": {}
+}
+```
+
+Example of using the TS client:
+
+```tsx
+const voucherWithdrawTxResult = await voucherContractClient.sendNft(
+  neutronUserAddress,
+  {
+    token_id: tokenId,
+    contract: withdrawalManagerContractAddress,
+    msg: Buffer.from(
+      JSON.stringify({
+        withdraw: {},
+      })
+    ).toString("base64"),
+  }
+);
+```
+
+### LSM Share Bond
+
+> At the moment of writing, LSM is only available on Cosmos Hub.
+
+Liquidity Staking Module (LSM) allows users to transfer their staked assets from one address to another without unstaking them. To learn more about it, visit the repository: https://github.com/iqlusioninc/liquidity-staking-module.
+
+To achieve that, LSM mints special tokens that represent staked assets ownership. One can "tokenize" their staked assets and by sending them transfer an ownership to other user or protocol. These tokens are called "LSM shares" for simplicity.
+
+Each LSM share is represented by a unique denom. It means that one can send a half of the share to someone but two different shares aren't fungible with each other.
+
+Drop can accept such shares and mint dASSET based on them. I.e., if one has already natively staked tokens (delegated to a validator), they can tokenize their shares, IBC transfer them to Neutron and "stake" with Drop the same way as ASSET itself.
+
+#### Message for tokenized share creation:
+
+```json
+{
+  "@type": "/cosmos.staking.v1beta1.MsgTokenizeShares",
+  "delegator_address": "cosmos_delegator_address",
+  "validator_address": "cosmosvaloper_validator_address_to_whom_you_staked",
+  "amount": {
+    "denom": "denom",
+    "amount": "1234"
+  },
+  "tokenized_share_owner": "cosmos_tokenized_share_owner"
+}
+```
+
+In order to reveal how many tokens you staked with validators from certain delegator use this query:
+
+```bash
+gaiad q staking delegations cosmos_delegator_address --node $THETA_TESTNET_NODE_ADDRESS
+```
+
+Also, please notice, that `validator_address` to whom you want delegate your tokens should be taken from our _white list_. To get this white list you need to query it from validators_set_contract. You can take it's address from core contract `config` query. In order to get validator set from given contract you need to query `validators` method.
```

Comment:
Can you please add more details with examples

Reply:
Does "with examples" mean direct link to the instance on celat.one with inlined query?